packages/members-api/lib/repositories/product.js

```diff
@@ -47,6 +52,27 @@ function validatePrice(price) {
     }
 }
 
+/**
+ * @param {BenefitInput} benefit
+ */
+function validateBenefit(benefit) {
+    if (!benefit.name) {
+        throw new ValidationError({
+            message: tpl(messages.benefitNameRequired)
+        });
+    }
+    if (typeof benefit.name !== 'string') {
```

Comment:
This won't get triggered if `benefit.name` is `null` or `undefined` because of the falsy check above.

It might be worth removing the check above? 🤔